content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -144,3 +146,11 @@ the CEL expressions that failed, respectively.
 The `validationActions` shows what actions were taken for this validation failure.
 See [Validating Admission Policy](/docs/reference/access-authn-authz/validating-admission-policy/)
 for more details about `validationActions`.
+
+## Event User.Extra Fields
+
+### authentication.kubernetes.io/credential-id
+
+{{< feature-state feature_gate_name="ServiceAccountTokenJTI" >}}
+
+This field contains the credential ID that was used to authenticate this request.  For service account tokens, this is the JTI claim embedded in the token, in the form `JTI=<jti>`.  For X.509 certificates, this is the SHA-256 fingerprint of the certificate, in the form `X509SHA256=<hexadecimal-encoded-hash>`.
```

Comment:
hoist this above the feature gate bit that only gates token ids?

```
This field contains the credential ID that was used to authenticate this request.

For X.509 certificates, this is the SHA-256 fingerprint of the certificate, in the form `X509SHA256=<hexadecimal-encoded-hash>`.
```

Reply:
Done